openshift_metrics/merge.py

```diff
@@ -145,23 +146,24 @@ def main():
     utils.write_metrics_by_pod(condensed_metrics_dict, pod_report_file, ignore_hours)
 
     if args.upload_to_s3:
+        bucket_name = os.environ.get("S3_INVOICE_BUCKET", "nerc-invoicing")
```

Comment:
Why an environment variable instead of a command line option (because all the other settings are options)?

Reply:
@larsks all S3 settings are environment variables (the key and the secret). No other good reason though.